plugins/KikiandHat3ditems/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+	"author": "KikiCreations and Hatfluppyclouds",
+	"id": "93114518-6866-4fbb-bfea-fb30fcea370e",
+	"version": "1.0.0",
+	"name": "KikiCreations and Hatfluppyclouds 3D Items",
+	"link": "https://www.youtube.com/channel/UCOt9hqBRhSvR3BCLhXdar1Q",
+	"description": "A plugin that adds a 3d item template",
+	"api_version": 1,
+	"tags": ["Presets"],
+	"releaseTimestamp": 1620567629002
```

Comment:
Please remove this property. The "releaseTimestamp" is supposed to get added by our automation upon merging the PR, otherwise it is inaccurate ^^